test/dlc_tests/benchmark/ec2/pytorch/training/test_performance_pytorch_training.py

```diff
@@ -63,6 +66,14 @@ def test_performance_pytorch_gpu_imagenet(pytorch_training, ec2_connection, gpu_
         ec2_connection, pytorch_training, PT_PERFORMANCE_TRAINING_GPU_IMAGENET_CMD
     )
 
+@pytest.mark.integration("inductor")
+@pytest.mark.parametrize("ec2_instance_ami", [PT_GPU_PY3_BENCHMARK_IMAGENET_AMI_US_WEST_2], indirect=True)
+@pytest.mark.parametrize("ec2_instance_type", [PT_EC2_GPU_INDUCTOR_INSTANCE_TYPES], indirect=True)
+def test_performance_pytorch_gpu_inductor(pytorch_training, ec2_connection, gpu_only, py3_only):
```

Comment:
Separate the inductor tests to a new test plan file called "test_performance_pytorch_inductor_training". This will allow us to specify a CODEOWNER for all files with *inductor* in them